pkg/cmd/version/version_test.go

```diff
@@ -290,6 +293,13 @@ func TestGetVersions(t *testing.T) {
 		userProvidedNamespace: "test",
 		deployment:            []*v1.Deployment{pipelineDeployment, triggersDeployment, dashboardDeployment},
 		goldenFile:            true,
+	}, {
+		name:                  "deployment with pipeline, triggers, dashboard and operator installed",
+		namespace:             "test",
+		userProvidedNamespace: "test",
+		deployment:            []*v1.Deployment{pipelineDeployment, triggersDeployment, dashboardDeployment},
```

Comment:
how about we provide the configmap for the remaining components as well instead of using deployments?

Reply:
yes but I was thinking to add a follow up pr which adds some more test using config map